File: core/src/com/inkostilation/pong/commands/response/ResponseEmptyCommand.java
package com.inkostilation.pong.commands.response;

import com.inkostilation.pong.commands.AbstractResponseCommand;
import com.inkostilation.pong.exceptions.NoEngineException;

import java.io.IOException;

public class ResponseEmptyCommand extends AbstractResponseCommand {
    @Override
    public void execute() throws IOException, NoEngineException {

    }
}

File: core/src/com/inkostilation/pong/commands/response/ResponseEmptyCommand.kt
package com.inkostilation.pong.commands.response

import com.inkostilation.pong.commands.AbstractResponseCommand
import com.inkostilation.pong.exceptions.NoEngineException
import java.io.IOException

class ResponseEmptyCommand : AbstractResponseCommand() {
    @Throws(IOException::class, NoEngineException::class)
    override fun execute() {
    }
}